packages/vsx-registry/src/browser/vsx-extensions-view-container.ts

```diff
@@ -43,7 +43,7 @@ export class VSXExtensionsViewContainer extends ViewContainer {
 
         this.setTitleOptions({
             label: VSXExtensionsViewContainer.LABEL,
-            iconClass: codicon('extensions'),
+            iconClass: 'theia-vsx-extensions-icon',
```

Comment:
@msujew any reason we want to revert the icon for the `extensions-view`?

If we wanted to add an example in the codebase for the use of a non-codicon `iconClass` we can use the following sample:

https://github.com/eclipse-theia/theia/blob/1738469b045934201215c1752965d4d45f4eba1d/examples/api-samples/src/browser/view/sample-unclosable-view.tsx#L33

Reply:
This is just a test commit, though using the old window icon works as well for a more permanent regression test 👍

Reply:
Hm, looking through it, it uses a FontAwesome icon, which won't show a regression. Only svg based icons were affected by the regression.

Reply:
> Hm, looking through it, it uses a FontAwesome icon, which won't show a regression. Only svg based icons were affected by the regression.

I just mean we can modify the one in the sample to now use an svg icon to test for backwards compatibility, not a codicon or font-awesome one :+1:

Reply:
Sure, makes sense